src/main/scala/io/iohk/ethereum/crypto/ECDSASignature.scala

```diff
@@ -72,6 +74,19 @@ object ECDSASignature {
     else s
   }
 
+  def sign(message: Array[Byte], keyPair: AsymmetricCipherKeyPair): ECDSASignature = {
```

Comment:
I think `sign` does belong here. Also what if we add differenet signature schemes? Anyway, checkout task EC-157